girder_worker/plugins/docker/executor.py

```diff
@@ -157,20 +184,24 @@ def run(task, inputs, outputs, task_inputs, task_outputs, **kwargs):
             outputs['_stdout']['script_data'] = ''
             print_stdout = False
 
-    command = ['docker', 'run', '-u', str(os.getuid())]
+    uid = str(os.getuid())
+    gid = str(os.getgid())
+    command = ['docker', 'run', '-u', '%s:%s' % (uid, gid)]
 
     if tempdir:
-        command += ['-v', tempdir + ':/data']
+        command += ['-v', '%s:%s' % (tempdir, DATA_VOLUME)]
 
-    if 'entrypoint' in task:
-        command += ['--entrypoint', task['entrypoint']]
+    scripts_dir = os.path.join(
+        os.path.abspath(os.path.dirname(__file__)), 'scripts')
+    command += ['-v', '%s:%s:%s' % (scripts_dir, SCRIPTS_VOLUME, 'ro')]
```

Comment:
You could directly mount the file as detailed in https://github.com/girder/girder_worker/issues/24#issuecomment-212567820

That way you have full control on the path